tests/test_scene_rendering/opengl/test_cli_flags_opengl.py

```diff
@@ -36,22 +36,21 @@ def test_basic_scene_with_default_values(tmp_path, manim_cfg_file, simple_scenes
     assert exit_code == 0, err
 
 
-@pytest.mark.slow()
+@pytest.mark.slow
 def test_resolution_flag(tmp_path, manim_cfg_file, simple_scenes_path):
     scene_name = "NoAnimations"
+    # test different separators
     resolutions = [
-        (720, 480),
-        (1280, 720),
-        (1920, 1080),
-        (2560, 1440),
-        (3840, 2160),
-        (640, 480),
-        (800, 600),
+        (720, 480, ";"),
+        (1280, 720, ","),
+        (1920, 1080, "-"),
+        (2560, 1440, ";"),
+        # (3840, 2160, ","),
```

Comment:
Are these just left commented out to allow for local testing of these resolutions?

Reply:
Nope, I each one of these tests takes ages and do we really need that many different resolutions to be tested?

Reply:
I don’t think so, think we can just remove the commented out ones ?

Reply:
Ups, I misunderstood your first comment. That's exactly why they're commented out and not deleted.